shared/actions/settings.js

```diff
@@ -403,6 +403,14 @@ function* processorProfile(_, action) {
 const rememberPassphrase = (_, action) =>
   RPCTypes.configSetRememberPassphraseRpcPromise({remember: action.payload.remember})
 
+const testPassphrase = (_, action) =>
```

Comment:
what do you think about this becoming `checkPassphrase` as well? there was some concern at CORE that it might be mistaken for a test code of some kind - I don't know if you share these. but feel free to leave it as is, I don't feel strongly about it